packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -121,22 +134,9 @@ export class ConnectionService {
       outOfBandId: outOfBandRecord.id,
       invitationDid,
       imageUrl: outOfBandInvitation.imageUrl,
+      threadId: connectionRequest.id,
     })
 
-    const { label, imageUrl, autoAcceptConnection } = config
-
-    const connectionRequest = new ConnectionRequestMessage({
-      label: label ?? this.config.label,
-      did: didDoc.id,
-      didDoc,
-      imageUrl: imageUrl ?? this.config.connectionImageUrl,
-    })
-
-    if (autoAcceptConnection !== undefined || autoAcceptConnection !== null) {
-      connectionRecord.autoAcceptConnection = config?.autoAcceptConnection
-    }
```

Comment:
It looks like we're losing this line

Reply:
This was probably a coding mistake. The condition `autoAcceptConnection !== undefined || autoAcceptConnection !== null` is always true. And the `connectionRecord.autoAcceptConnection` was already initialized with the same value `config?.autoAcceptConnection` before.

Reply:
You're right 👍